From 9bb962148672a3c20fb0119d7f71789b8ef3dc02 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 29 Jun 2021 19:31:34 -0700
Subject: [PATCH] include <utility> for std::forward

Fixes
../git/src/Log.hxx:121:42: error: no member named 'forward' in namespace 'std'
        LogFormat(LogLevel::ERROR, e, fmt, std::forward<Args>(args)...);

Upstream-Status: Submitted [https://github.com/MusicPlayerDaemon/MPD/pull/1201]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/Log.hxx                   | 1 +
 src/lib/expat/ExpatParser.hxx | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/Log.hxx b/src/Log.hxx
index a295ed9f6..87097b086 100644
--- a/src/Log.hxx
+++ b/src/Log.hxx
@@ -24,6 +24,7 @@
 #include "util/Compiler.h"
 
 #include <exception>
+#include <utility>
 
 class Domain;
 
diff --git a/src/lib/expat/ExpatParser.hxx b/src/lib/expat/ExpatParser.hxx
index e1723d109..4b5221e89 100644
--- a/src/lib/expat/ExpatParser.hxx
+++ b/src/lib/expat/ExpatParser.hxx
@@ -25,6 +25,7 @@
 #include <expat.h>
 
 #include <stdexcept>
+#include <utility>
 
 class InputStream;
 
-- 
2.32.0

